docs/source/guides/auth.mdx

```diff
@@ -243,6 +243,34 @@ Before continuing, you need to set up the Auth0 client to accept an additional c
 
 </ExpansionPanel>
 
+## Step 5: Make requests with an MCP Client (Claude)
+
+In order to get the full experience of using and MCP server authorization, you'll want to use an LLM of some sort.
+Anthropic's Claude provides an accessible method of connecting to an MCP server, through [their Custom Connector feature](https://support.anthropic.com/en/articles/11175166-getting-started-with-custom-connectors-using-remote-mcp),
+which is available to paid Claude subscribers. If you don't have a paid subscription, or want to try Claude out without deploying your MCP Server behind HTTPS (a requirement for Custom Connectors),
+follow these steps to get Claude Desktop working locally:
+
+1. Navigate to **Settings > Developer**.
+1. Click **Edit Config** and open `claude_desktop_config.json` in your text editor.
+1. If there are any JSON keys other than `mcpServers` in the file, leave them alone, but make sure you add your local MCP Server to that key:
+  ```json title="claude_desktop_config.json" {3-11}
+  {
+    "mcpServers": {
+      "theSpaceDevs": {
+        "command": "npx",
+        "args": [
+          "mcp-remote",
+          "http://127.0.0.1:5000/mcp",
+          "--transport",
+          "http-first"
+        ]
+      }
+    }
+  }
+  ```
+  1. Quit Claude Desktop and open it back up. Claude will open a browser window for the OAuth flow and turn on the operations configured for your MCP Server.
```

Comment:
This is where things start going weird. I got it to work once and made it through the next step, so I've pushed this up, but it's pretty rough. We can sync up on this after the holiday.